lib/grammers-client/src/types/message.rs

```diff
@@ -476,10 +476,9 @@ impl Message {
         self.client
             .get_messages_by_id(&self.chat(), &[self.msg.id])
             .await?
-            .pop()
-            .unwrap()
+            .get(0)
```

Comment:
The error points this should've been kept unchanged.

Reply:
oh, ok